### PR TITLE
table-attributes: minor fixes

### PR DESCRIPTION
**Pull request by dveeden**
### What is changed, added or deleted? (Required)

Small fixes for the table attributes document

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

This PR is in line with https://github.com/pingcap/docs-cn/pull/7627.

### CODE REVIEW

**Comment by dveeden** [comment]
I would suggest to uppercase SQL keywords. (`ALTER TABLE` instead of `alter table`), but I don't think we have a standard for that.

File: table-attributes.md [hunk 1]
```diff
@@ -57,12 +57,14 @@ alter table t partition p attributes[=]'key=value1';
...
 In this case, `key=value1` is the attribute that actually takes effect on the `p1` partition.
 
-## Control the Region merge bahavior using table attributes
+## Control the Region merge behavior using table attributes
 
 ### User scenarios
 
 If there is a write hotspot or read hotspot, you can use table attributes to control the Region merge behavior. You can first add the `merge_option` attribute to a table or partition and then set its value to `deny`. The two scenarios are as follows.
 
+This is only part of how to deal with hotspots, see [Troubleshoot Hotspot Issues](/troubleshoot-hot-spot-issues.md) for more information.
```
Comment (en-jin19) [inline table-attributes.md:66]:
```suggestion
This is only part of how to deal with hotspots. For more information, refer to [Troubleshoot Hotspot Issues](/troubleshoot-hot-spot-issues.md).
```

**Comment by dveeden** [comment]
```
sql> SHOW CREATE TABLE t\G
*************************** 1. row ***************************
       Table: t
Create Table: CREATE TABLE `t` (
  `id` int(11) NOT NULL,
  PRIMARY KEY (`id`) /*T![clustered_index] CLUSTERED */
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin
1 row in set (0.0005 sec)

sql> ALTER TABLE t attributes = 'merge_option=deny';
Query OK, 0 rows affected (0.0045 sec)

sql> SHOW CREATE TABLE t\G
*************************** 1. row ***************************
       Table: t
Create Table: CREATE TABLE `t` (
  `id` int(11) NOT NULL,
  PRIMARY KEY (`id`) /*T![clustered_index] CLUSTERED */
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin
1 row in set (0.0003 sec)
```
I would have expected the attributes to be visible in the output of `SHOW CREATE TABLE...`, but it is not. Unless the feature gets changed this should be stated more explicitly in the docs.

**Comment by dveeden** [comment]
```
sql> select * from information_schema.attributes;
+---------------+-----------+---------------------+------------------------------------------------------------------------------+
| ID            | TYPE      | ATTRIBUTES          | RANGES                                                                       |
+---------------+-----------+---------------------+------------------------------------------------------------------------------+
| schema/test/t | key-range | "merge_option=deny" | [7480000000000000ff395f720000000000fa, 7480000000000000ff3a5f720000000000fa] |
+---------------+-----------+---------------------+------------------------------------------------------------------------------+
1 row in set (0.0004 sec)

sql> ALTER TABLE t attributes = 'foo=bar';
Query OK, 0 rows affected (0.0042 sec)

sql> select * from information_schema.attributes;
+---------------+-----------+------------+------------------------------------------------------------------------------+
| ID            | TYPE      | ATTRIBUTES | RANGES                                                                       |
+---------------+-----------+------------+------------------------------------------------------------------------------+
| schema/test/t | key-range | "foo=bar"  | [7480000000000000ff395f720000000000fa, 7480000000000000ff3a5f720000000000fa] |
+---------------+-----------+------------+------------------------------------------------------------------------------+
1 row in set (0.0004 sec)
```
Setting attributes wipes out previously set attributes. Not sure if the docs need to state this more explicitly.

File: table-attributes.md [hunk 2]
```diff
@@ -57,12 +57,14 @@ alter table t partition p attributes[=]'key=value1';
...
 In this case, `key=value1` is the attribute that actually takes effect on the `p1` partition.
 
-## Control the Region merge bahavior using table attributes
+## Control the Region merge behavior using table attributes
 
 ### User scenarios
 
 If there is a write hotspot or read hotspot, you can use table attributes to control the Region merge behavior. You can first add the `merge_option` attribute to a table or partition and then set its value to `deny`. The two scenarios are as follows.
 
+This is only part of how to deal with hotspots. For more information, refer to [Troubleshoot Hotspot Issues](/troubleshoot-hot-spot-issues.md).
```
Comment (TomShawn) [inline table-attributes.md:66]:
```suggestion
This is only part of how to deal with hotspots. For more information, refer to [Troubleshoot Hotspot Issues](/troubleshoot-hot-spot-issues.md). 
```

**Review by TomShawn — APPROVED** [review]

**Comment by rleungx** [comment]
> ```
> sql> SHOW CREATE TABLE t\G
> *************************** 1. row ***************************
>        Table: t
> Create Table: CREATE TABLE `t` (
>   `id` int(11) NOT NULL,
>   PRIMARY KEY (`id`) /*T![clustered_index] CLUSTERED */
> ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin
> 1 row in set (0.0005 sec)
> 
> sql> ALTER TABLE t attributes = 'merge_option=deny';
> Query OK, 0 rows affected (0.0045 sec)
> 
> sql> SHOW CREATE TABLE t\G
> *************************** 1. row ***************************
>        Table: t
> Create Table: CREATE TABLE `t` (
>   `id` int(11) NOT NULL,
>   PRIMARY KEY (`id`) /*T![clustered_index] CLUSTERED */
> ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin
> 1 row in set (0.0003 sec)
> ```
> 
> I would have expected the attributes to be visible in the output of `SHOW CREATE TABLE...`, but it is not. Unless the feature gets changed this should be stated more explicitly in the docs.

In the current implementation, we don't support `SHOW CREATE TABLE`. So you can only query information_schema.attributes to get the details of the attributes which is mentioned in this doc.

**Comment by rleungx** [comment]
> 

Yes, it's better to let the user know it.

File: table-attributes.md [hunk 3]
```diff
@@ -19,50 +19,52 @@ The table attribute is in the form of `key=value`. Multiple attributes are separ
...
```
Comment (en-jin19) [inline table-attributes.md:66]: @dveeden how about moving this part above "### Usage" and changing it as a "Note"?

Comment (dveeden) [reply table-attributes.md]: Done.

File: table-attributes.md [hunk 4]
```diff
@@ -5,12 +5,13 @@ summary: Learn how to use the table attribute feature of TiDB.
...
+The Table Attributes feature is introduced in TiDB v5.3.0. Using this feature, you can add specific attributes to a table or partition to perform the operations corresponding to the attributes. For example, you can use table attributes to control the Region merge behavior.
 
 > **Note:**
 >
 > - Currently, TiDB only supports adding the `merge_option` attribute to a table or partition to control the Region merge behavior.
 > - When you use TiDB Binlog or TiCDC to perform replication or use BR to perform incremental backup, the replication or backup operations skip the DDL statement that sets table attributes. To use table attributes in the downstream or in the backup cluster, you need to manually execute the DDL statement in the downstream or in the backup cluster.
+> - The `merge_option` attribute is only part of how to deal with hotspots. For more information, refer to [Troubleshoot Hotspot Issues](/troubleshoot-hot-spot-issues.md).
```
Comment (en-jin19) [inline table-attributes.md:14]:
```suggestion
> - Currently, TiDB only supports adding the `merge_option` attribute to a table or partition to control the Region merge behavior. The `merge_option` attribute is only part of how to deal with hotspots. For more information, refer to [Troubleshoot Hotspot Issues](/troubleshoot-hot-spot-issues.md).
> - When you use TiDB Binlog or TiCDC to perform replication or use BR to perform incremental backup, the replication or backup operations skip the DDL statement that sets table attributes. To use table attributes in the downstream or in the backup cluster, you need to manually execute the DDL statement in the downstream or in the backup cluster.
```

File: table-attributes.md [hunk 5]
```diff
@@ -19,45 +20,45 @@ The table attribute is in the form of `key=value`. Multiple attributes are separ
...
     ```sql
-    select * from information_schema.attributes where id='schema/t[/p]';
+    SELECT * FROM information_schema.attributes WHERE id='schema/t[/p]';
     ```
 
 + See all tables and partitions that have a specific attribute:
 
     ```sql
-    select * from information_schema.attributes where attributes like '%key%';
+    SELECT * FROM information_schema.attributes WHERE attributes LIKE '%key%';
```
Comment (en-jin19) [inline table-attributes.md:47]:
```suggestion
    SELECT * FROM information_schema.attributes WHERE ATTRIBUTES LIKE '%key%';
```

File: table-attributes.md [hunk 6]
```diff
@@ -76,49 +77,49 @@ Suppose that in a read-only scenario, you try to reduce the periodic read hotspo
...
     ```sql
-    alter table t attributes[=]'merge_option=allow';
+    ALTER TABLE t ATTRIBUTES[=]'merge_option=allow';
     ```
 
 + Reset attributes of a table:
 
     ```sql
-    alter table t attributes[=]default；
+    ALTER TABLE t ATTRIBUTES[=]default；
```
Comment (en-jin19) [inline table-attributes.md:92]: should "default" be in upper cases? @dveeden

File: table-attributes.md [hunk 7]
```diff
@@ -76,49 +77,49 @@ Suppose that in a read-only scenario, you try to reduce the periodic read hotspo
...
     ```
 
 + Prevent the Regions of a partition from merging:
 
     ```sql
-    alter table t partition p attributes[=]'merge_option=deny';
+    ALTER TABLE t PARTITION p attributes[=]'merge_option=deny';
```
Comment (en-jin19) [inline table-attributes.md:98]:
```suggestion
    ALTER TABLE t PARTITION p ATTRIBUTES[=]'merge_option=deny';
```

File: table-attributes.md [hunk 8]
```diff
@@ -76,49 +77,49 @@ Suppose that in a read-only scenario, you try to reduce the periodic read hotspo
...
     ```
 
 + Allow merging Regions belonging to a partition:
 
     ```sql
-    alter table t partition p attributes[=]'merge_option=allow';
+    ALTER TABLE t PARTITION p attributes[=]'merge_option=allow';
```
Comment (en-jin19) [inline table-attributes.md:104]:
```suggestion
    ALTER TABLE t PARTITION p ATTRIBUTES[=]'merge_option=allow';
```

File: table-attributes.md [hunk 9]
```diff
@@ -76,49 +77,49 @@ Suppose that in a read-only scenario, you try to reduce the periodic read hotspo
...
     ```
 
 + See all tables or partitions configured the `merge_option` attribution:
 
     ```sql
-    select * from information_schema.attributes where attributes like '%merge_option%';
+    SELECT * FROM information_schema.attributes WHERE attributes LIKE '%merge_option%';
```
Comment (en-jin19) [inline table-attributes.md:110]:
```suggestion
    SELECT * FROM information_schema.attributes WHERE ATTRIBUTES LIKE '%merge_option%';
```

File: table-attributes.md (hunk 6 above)
Comment (dveeden) [reply table-attributes.md:91]: Yes:

- https://docs.pingcap.com/tidb/stable/keywords#D
- https://github.com/pingcap/tidb/blob/e65f548a40acedd51caa67bed9ee44f51baf852e/parser/parser.y#L113

```suggestion
    ALTER TABLE t ATTRIBUTES[=]DEFAULT；
```

**Comment by en-jin19** [comment]
hi @dveeden, to make sure the documents in Chinese and English have the same SQL statements, I have updated the SQL statements in this PR: removed some [=] and added blanks. PTAL, thank you!

**Review by en-jin19 — APPROVED** [review]
LGTM

**Review by rleungx — APPROVED** [review]

**Pull request merged**